diff --git a/src/video_core/CMakeLists.txt b/src/video_core/CMakeLists.txt
index a021d61f5..15c5dedfb 100644
--- a/src/video_core/CMakeLists.txt
+++ b/src/video_core/CMakeLists.txt
@@ -310,7 +310,6 @@ else()
         -Wno-error=sign-conversion
         -Werror=pessimizing-move
         -Werror=redundant-move
-        -Werror=shadow
         -Werror=switch
         -Werror=type-limits
         -Werror=unused-variable
